src/main/java/tasks/Deadlines.java

```diff
@@ -0,0 +1,46 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
```

Comment:
I think Deadline would be a better class name than Deadlines. For Events as well.